docs/DeveloperGuide.adoc

```diff
@@ -795,6 +795,64 @@ Private contact detail
 A contact detail that is not meant to be shared with others
 ....
 
+
+[[contact-information]]
+Contact information of a person
```

Comment:
Capitalise the title "Contact Information of a Person"

Reply:
I was following the earlier style so I don't really know if it should be capital or not. However, in the HTML appearence, the text displayed is the second line, not the text within the bracket.